check_mongodb.py

```diff
@@ -18,18 +18,23 @@
 #   - @Andor on github
 #   - Steven Richards - Captainkrtek on github
 #   - Max Vernimmen
+#   - @burdandrei Added CloudWatch monitoring for Mongodb
 #
 # USAGE
 #
 # See the README.md
 #
 
+
 import sys
 import time
+import datetime
 import optparse
 import textwrap
 import re
 import os
+import commands
+from boto.ec2.cloudwatch import CloudWatchConnection
```

Comment:
this will break the plugin for a lot of people that don't have boto installed.. we should wrap it in a try/except to enable the check later on or not